src/pages/patterns/tables/basic-formatting.mdx

```diff
@@ -0,0 +1,441 @@
+
+<CodeExample code={TableVerticalScrollingCode}>
+  <TableVerticalScrolling />
+</CodeExample>
+
+##### Virtualization
+
+Virtualization means what you see in the browser is what is in the DOM. When something scrolls out of view, it gets
+removed from the page DOM. When something scrolls into view, it is added to the DOM.
+
+- Use [react-window](https://github.com/bvaughn/react-window) to efficiently render large data sets
+
+import TableVirtualized from '../examples/tables/TableVirtualized.tsx';
+import TableVirtualizedCode from '!!raw-loader!../examples/tables/TableVirtualized.tsx';
+
+<CodeExample code={TableVirtualizedCode}>
+  <TableVirtualized />
+</CodeExample>
+
+#### Sorting
+
+Sorting is implemented by selecting arrows in table headers.
+
+- Clicking the header text will sort by ascending order (A→Z, 1→9, Date and Time, Priority (High→Low))
+- Clicking again will reverse the order
+- Clicking a third time will remove sorting
+
+If any ordering applied to a table including default ordering, the sorting icon should be present. This indicates how
+the table is being ordered and also creates a better affordance for the control.
+
+import TableSort from '../examples/tables/TableSort.tsx';
+import TableSortCode from '!!raw-loader!../examples/tables/TableSort.tsx';
+
+<CodeExample code={TableSortCode}>
+  <TableSort />
+</CodeExample>
+
+#### Draggable
+
+T​​o have row reordering, you can apply drag-and-drop functionality with [react-beautiful-dnd](https://github.com/atlassian/react-beautiful-dnd).
+For accessibility support, please [read the React beautiful dnd guidelines](https://github.com/atlassian/react-beautiful-dnd/blob/master/docs/about/accessibility.md).
+
+import TableDraggable from '../examples/tables/TableDraggable.tsx';
+import TableDraggableCode from '!!raw-loader!../examples/tables/TableDraggable.tsx';
+
+<CodeExample code={TableDraggableCode}>
+  <TableDraggable />
+</CodeExample>
+
+#### Table caption
+
+The [Caption](/components/table#table-caption) is the title of a table.
+
+A table must have a [Caption](/components/table#caption) or aria-describedby attribute that identifies the element which
+titles or describes the table
+
+import TableCaption from '../examples/tables/TableCaption.tsx';
+import TableCaptionCode from '!!raw-loader!../examples/tables/TableCaption.tsx';
+
+<CodeExample code={TableCaptionCode}>
+  <TableCaption />
+</CodeExample>
+
+#### Anchors
+
+Use an [Anchor](/components/anchor) for data cell text elements that link elsewhere.
+
+![Anchors](figma:patterns-tables-basic-formatting-anchors.png)
+
+1. These anchors within a table typically navigate the user to a new page
+1. If you want to open a link in an external window or tab, use an [External Anchor](/components/anchor#external-anchor)
+
+#### Date and Time
+
+Follow our [date and time formatting](/content/grammar#date-and-time) guidelines.
+
+![Date and Time](figma:patterns-tables-basic-formatting-date-time.png)
```

Comment:
Reflects the [existing Garden content design guidelines for date and time](https://garden.zendesk.com/content/grammar#date-and-time)

Reply:
Hm, it does look strange in the image. Can we change that to an actual example like the other entries instead of exposing the formula? I'm concerned other folks will also be confused. Since the content guidelines are linked, the formula is there for folks to refer back to.

Reply:
@diana-oum Good point. I'll update the screenshots ccing @hzhu